src/Templates/parts/nav.twig

```diff
@@ -10,6 +10,7 @@
       </button>
       {% endif %}
       <a class="navbar-brand" href="{{baseurl}}">MyRadio</a>
+      <a class="navbar-brand" href="{{homeurl}}">ury.org.uk</a>
```

Comment:
@LordAro I did mention that, I can also create a var for the name.

@AJDurant I will sort that now